### DISCUSSION TITLE
Kanban for Primevue Apollo as seen in PrimeNG

### DISCUSSION THREAD

**Discussion by sllawcj** [primefaces/community#119] [PrimeVue Templates] [answered]
Hello, is there any plan or roadmap to include a kanban app in the primevue apollo template as seen in the primeNG apollo template?

Thanks!

**Comment by sezisfurkan** [comment, marked as answer]
Hi,
 yes, it's in our plans, and our priority is to add the necessary functions to the core for this. Currently, there's no definite timeline, but when it becomes available, we will announce it.
